db/migrate/20190204150122_add_reading_time_to_facts_edition.rb

```diff
@@ -0,0 +1,5 @@
+class AddReadingTimeToFactsEdition < ActiveRecord::Migration[5.2]
+  def change
+    add_column :facts_editions, :reading_time, :float
```

Comment:
Why use float for discrete values? - I assume the smallest resolution we care about is per second, which we could use an integer - Or could we use the Postgres interval type?

Reply:
We always need to round up to 1 minutes if the actual reading time is less than that (from conversation with PA and CD) but I'm now thinking just storing as seconds in integer is a better option. Will rework when I get a chance after 2ndline.

Reply:
Stored as minutes in integer.